### PR TITLE
Add Mouse Wheel support for named colorpicker 

### PR DESCRIPTION
**Pull request by ErlendRovik**
As the title says, add mouse wheel support for actual canvas, as a complement to s rollbar.

### CODE REVIEW

**Comment by ErlendRovik** [comment]
Documentation: Added long to IsStart (DragCallback) as it was missing
Fixed: a bug in my ColorPicker MouseWheel commit as it failed if you switched views (tab) and went back to Named color panel.

**Comment by t-harter** [comment]
I know both changes are small, but please create a separate pull request for documentation changes if they are not directly related to the changes in the IDE code.

File: PureBasicIDE/ColorPicker.pb [hunk 1]
```diff
@@ -1114,6 +1114,19 @@ Procedure ColorPicker_Name_Event(*Entry.ColorPickerData, Gadget, Type)
       EndIf
       
     Case #GADGET_Color_Canvas1
+
+      If Type = #PB_EventType_MouseWheel
+        *Entry\First-GetGadgetAttribute(#GADGET_Color_Canvas1,#PB_Canvas_WheelDelta)
```
Comment (t-harter) [inline PureBasicIDE/ColorPicker.pb:1119]: Please add spaces between individual expression parts and between procedure parameters to improve readability. I.e.:
```
If *Entry\First > *Palette\Count - GetGadgetAttribute(#GADGET_Color_Scroll, #PB_ScrollBar_PageLength)
```

Comment (ErlendRovik) [reply PureBasicIDE/ColorPicker.pb:1119]: Right :-) , I'm trying to get the hang of this Github stuff ;-)

**Pull request closed without merging**